src/components/MoneyReportHeader.tsx

```diff
@@ -346,6 +346,9 @@ function MoneyReportHeader({policy, report: moneyRequestReport, transactionThrea
         if (hasScanningReceipt) {
             return {icon: getStatusIcon(Expensicons.ReceiptScan), description: translate('iou.receiptScanInProgressDescription')};
         }
+        if (isEmpty) {
+            return {icon: getStatusIcon(Expensicons.Hourglass), description: 'Waiting for you to add expense(s)'};
```

Comment:
I'm not sure if this should be done this way or as the expected optimistic next step. I did it this way because it was easier to implement and didn't require changes in Onyx.